maps/CEVEris/_map_data_Eris.dm

```diff
@@ -203,28 +206,27 @@ ADMIN_VERB_ADD(/client/proc/test_MD, R_DEBUG, null)
 	var/is_accessable_level = TRUE // Prob modifier for random access (space travelling)
 	var/generate_asteroid= FALSE
 	var/tmp/z_level
+	var/height = -1	///< The number of Z-Levels in the map.
```

Comment:
Why default height is -1?

Reply:
This is for no one forgot to set the height ont he map. It's asserted at the New()

Reply:
Are you sure? It looks like it was asserted at new, but you've deleted this ? https://i.imgur.com/cjpEcXw.png